backend/tests/middlewares/authMiddleware.test.js

```
@@ -0,0 +1,117 @@
+const jwt = require('jsonwebtoken');
+const { AppDataSource } = require('../../src/config/database');
+const { verifyToken, requireRole } = require('../../src/middlewares/authMiddleware');
+
+jest.mock('jsonwebtoken');
+jest.mock('../../src/config/database', () => ({
+  AppDataSource: { getRepository: jest.fn() }
+}));
+
+describe('authMiddleware', () => {
+    let mockReq, mockRes, mockNext, userRepository;
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)